### PR TITLE
Sanitizer

### LINKED ISSUE #170: Update pre-built image
The goal of this issue is to put together a wish list for the next dtk-stack. Please feel free to edit.

### PR DESCRIPTION
**Pull request by Rombur**
connects to #164 
closes #170

### CODE REVIEW

File: docker/Dockerfile_stack [hunk 1]
```diff
@@ -24,15 +24,20 @@ RUN apt-get update && apt-get install -y \
         libatlas-base-dev \
         zlib1g-dev \
         python2.7-dev \
+        subversion \
```
Comment (dalg24) [inline docker/Dockerfile_stack]: Want me to install CVS too?

File: docker/Dockerfile_stack [hunk 2]
```diff
@@ -24,15 +24,20 @@ RUN apt-get update && apt-get install -y \
...
     wget -q -O - http://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
     apt-get update && apt-get install -y \
+        clang-3.9 \
         clang-format-3.9 \
         && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+RUN ln -s /usr/bin/clang-3.9 /usr/bin/clang
+RUN ln -s /usr/bin/clang++-3.9 /usr/bin/clang++
```
Comment (dalg24) [inline docker/Dockerfile_stack]: Please move these to `/usr/local/bin`.

File: docker/Dockerfile_stack [hunk 3]
```diff
@@ -45,6 +50,14 @@ RUN mkdir -p ${PREFIX} && \
     mkdir build && \
     mkdir install
 
+# Install OpenMP runtime library
+RUN cd ${PREFIX}/source && \
+    svn co http://llvm.org/svn/llvm-project/openmp/branches/release_39 openmp && \
```
Comment (dalg24) [inline docker/Dockerfile_stack]: Why not downloading http://releases.llvm.org/3.9.0/openmp-3.9.0.src.tar.xz?

Comment (dalg24) [inline docker/Dockerfile_stack]: Because it's the dev page.

File: docker/Dockerfile_stack (hunk 2 above)
Comment (dalg24) [inline docker/Dockerfile_stack]: I can't believe that apt is not taking care of the symlink...

Comment (aprokop) [inline docker/Dockerfile_stack]: https://bugs.launchpad.net/ubuntu/+source/llvm-toolchain-3.6/+bug/1566240

File: docker/Dockerfile_stack (hunk 3 above)
Comment (Rombur) [inline docker/Dockerfile_stack]: I live on the (bleeding-)edge

**Comment by Rombur** [comment]
Done

**Comment by dalg24** [comment]
Please push the new image to Docker Hub.  I will trigger a rebuild before I merge.

**Comment by Rombur** [comment]
Done

**Comment by dalg24** [comment]
I did minor edit/squashed/force push.

**Comment by Rombur** [comment]
LGTM

**Pull request merged**